apps/api/tests/testUtils.ts

```diff
@@ -49,14 +49,19 @@ export const initTestMigration = async (db: PostgresDb) => {
 	}
 };
 
+// Sets Number of Db Records to Seed @ start of test run using addInitialApplications
+export const numTestApplications = 20;
+// Used to bypass pagination impacting test results
+export const allRecordsPageSize = 2 * numTestApplications;
+
 export const addInitialApplications = async (db: PostgresDb) => {
```

Comment:
If you change these comments to tsdoc format they will be shown with intellisense where these are used.

In general, all our exported variables should have TSDocs.

Reply:
I will update this w/ the followup to [Collaborators PR](https://github.com/Pan-Canadian-Genome-Library/daco/pull/175)